aiohttp/http_parser.py

```diff
@@ -588,7 +588,7 @@ def parse_message(self, lines: List[bytes]) -> RawRequestMessage:
             url = URL(path, encoded=True)
             if url.scheme == "":
                 # not absolute-form
-                raise InvalidURLError(line)
```

Comment:
Should we use backslashreplace here too?

Reply:
Thanks for pointing this out.

Encoding with backslashreplace would leave us with a `bytes` that we need to `repr` for the error message, so I went with `surrogateescape` and reencode to `latin1.`